docs/architecture/dicts/spec_config.py

```diff
@@ -0,0 +1,20 @@
+
+
+import radical.utils as ru
+import spec_attribs  as a
+
+
+class CFG(ru.Config):
```

Comment:
I am not sure I like this naming. I see it is concise but even after reading the documentation just 5 minutes ago I cannot remember what it stands for.